### DISCUSSION TITLE
YI Firmware 9.x to 12.x

### DISCUSSION THREAD

**Discussion by cbroissa-github** [roleoroleo/yi-hack-Allwinner-v2#673] [Q&A]
Hello,
You think it’s interesting to upgrade my cameras with the actual YI FW ?
You know where I can find a change log ?
Thank you.

**Comment by roleoroleo** [comment]
> You think it’s interesting to upgrade my cameras with the actual YI FW ?

I think there are no big differences.
The biggest change is the AI component related to motion detections.
But I still have to figure out if it's an advantage or a disadvantage

> You know where I can find a change log ?

No.
